css/main.css

```diff
@@ -1,11 +1,6 @@
-body {
-	padding-top: 3em;
-	max-width: 45em;
```

Comment:
Have you tried without this instruction on a very wide screen? I'm not sure this shouldn't be part of the default readability settings for a text-based interface.